pages/common/git-fresh-branch.md

```diff
@@ -0,0 +1,9 @@
+# git fresh-branch
+
+> Creates an empty local branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md##git-fresh-branch>.
+
+- [c]reate an empty local branch:
```

Comment:
```suggestion
- Create an empty local branch:
```

By the way, do not use mnemonics if it isn't linked to short command parameters (i.e. `-c`)